pages/contribute/rule-feedback.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Give feedback on rules
+---
+
+Feedback on rules is encouraged on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
+
+If you "watch" the repository, you will get notifications when changes are happening.
```

Comment:
It might be helpful if "watch" links to something that explains how to do that.

Reply:
It might actually be possible to have a link to watch automatically.
(when new repos are created in my org, Github auto-watch me and send me a link to un-watch; if link to un-watch exists, I assume link to watch should exist)

Reply:
Added link to Propose Rules page.